app/Policies/BillingPolicy.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Policies;
+
+use App\Billing;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class BillingPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any billings.
+     *
+     * @param User $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
+    {
+        return true;
+    }
+
+    /**
+     * Determine whether the user can view the billing.
+     *
+     * @param User $user
+     * @param Billing $billing
+     * @return mixed
+     */
+    public function view(User $user, Billing $billing)
+    {
+        return $user->is($billing->owner);
```

Comment:
Fajnie się układa ta linia semantycznie - kwintesencja czystego kodu. 

Jeżeli miałoby to być czytlene dla każdego programisty to lepszym wyborem byłoby (niestety trochę mniej semantycznie):
 $user->equals($billing->owner)

Reply:
a nie czytelniejsze jest własnie "użytkownik jest właścicielem billingu" niż "użytkownik równa się właściciel billingu"?

Reply:
Jak najbardziej masz rację. Chodzi o to, że wiele języków/frameworków (poza PHPowych) wprowadza equals do takiej operacji. Choć equals w tym przypadku jest mniej semantyczne to jest bardziej naturalne w programowaniu. - tak jak rozmawialiśmy na slacku